### PR TITLE
Fix test_missing

### PR DESCRIPTION
**Pull request by markgoddard**

### CODE REVIEW

**Review by valeriupredoi — CHANGES_REQUESTED** [review]

**Comment by valeriupredoi** [comment]
@markgoddard cheers for your ongoing efforts :beer: Could I ask you for one thing plss - it'd be cool if you were a bit more verbose/explicit when naming your branches - we now have two branches one called `fix-missing-tests` and another `fix-test-missing` which is confusing :+1:

File: tests/test_missing.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def test_partially_missing_data(tmp_path):
 
 def test_missing(tmp_path):
     testfile = str(tmp_path / 'test_missing.nc')
-    r = dd.make_partially_missing_ncdata(testfile)
+    r = dd.make_missing_ncdata(testfile)
```
[the remark this answers is not in the document]
Comment (markgoddard) [reply tests/test_missing.py:41]: Before, test_missing was exactly the same as test_partially_missing_data, except for a different test file name (which has no impact on the test). I assumed it was a mistake, and that you wanted test both missing and partially missing data to align with the test name.

**Comment by valeriupredoi** [comment]
let's have an issue about that, and take it step by step - it's much better now that we have a decent number of contributors, have a look at this https://github.com/valeriupredoi/PyActiveStorage/issues/102 (PS note that I didn't delete the branch so we can always resurrect it in the right manner, but let's try be a bit more cleanly organized :smiley: )

**Comment by bnlawrence** [comment]
I'm reopening it until I find out. Too many things that all look the same.

**Comment by markgoddard** [comment]
I force pushed the branch with a more detailed commit message. This prevents us from reopening this PR, so I've created another: https://github.com/valeriupredoi/PyActiveStorage/pull/104

Sorry for the churn here and lack of detail. I thought this one was obvious, so didn't provide much info, but perhaps it's one of those things that's only obvious once you've seen it.

As for the other PRs in this area, sorry they've been a bit sloppy - my understanding of the underlying issue is evolving. I'll dig for a bit then open an issue.

**Comment by valeriupredoi** [comment]
many thanks, Mark! And absolutely understandable, it's also my way of working on a project that is fairly dogmatic and rules-based, so apologies if I came across like an accountant, but I reckon the more we stick to some PR opening/review rules, the easier the job is for us :)

**Pull request closed without merging**